src/lib/fauna-command.js

```diff
@@ -85,7 +85,12 @@ class FaunaCommand extends Command {
     return errorOut(err, 1)
   }
 
-  async getClient({ dbScope, role } = {}) {
+  getQueryStats(res) {
+    if (!res || !res.responseHeaders) return
+    console.log(res.responseHeaders)
```

Comment:
is this intended just to be a POC?

How can we get this out to the user of the client so we can consume the stats?